docs/config.md

```diff
@@ -197,6 +201,47 @@ names:
   commonNamePattern: "{host}.buoyant.io"
 ```
 
+#### Load Balancer
+
```

Comment:
Good point.  The default (p2c) comes from finagle so I'm a bit uneasy documenting something that could change out from under us.  But in this case I think it makes sense.